paper/onboarding_issue.md

```diff
@@ -59,12 +59,12 @@ Anyone who is interested in making use of the Water Survey of Canada data source
 -   Are there other R packages that accomplish the same thing? If so, how does
 yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?
 
-    - A search of the term "hydat" on github reveals two R packages that are provide similar (but not all the same) functionality:
-    - [HYDAT](https://github.com/CentreForHydrology/HYDAT): the original author is a contributor to this package and will not be further developing this package soon. 
-    -[hydatr](https://github.com/paleolimbot/hydatr) was develop at approximately the same time as `tidyhdyat`. 
+    - A search of the term "hydat" on github reveals two R packages that provide similar (but not all the same) functionality:
+    - [HYDAT](https://github.com/CentreForHydrology/HYDAT): the original author of **HYDAT** is a contributor to this package and will soon cease further development of **HYDAT**. 
+    -[hydatr](https://github.com/paleolimbot/hydatr) was developed at approximately the same time as `tidyhdyat`. 
     - `tidyhydat` documentation is more thorough than both packages.
     - Through the realtime functions `tidyhydat` provides greater functionality than both packages (webservice for both; datamart for `hydatr`)
```

Comment:
I'm a bit unclear what you mean here (webservice for both; datamart for `hydatr`)